tests/sqlite3/test-sqlite3.cpp

```diff
@@ -869,6 +872,31 @@ TEST_CASE("SQLite std::tm bind", "[sqlite][std-tm-bind]")
     CHECK(std::mktime(&result.front()) == std::mktime(&datetime));
 }
 
+struct integer_table_creator : table_creator_base
+{
+    integer_table_creator(soci::session &sql)
+        : table_creator_base(sql)
+    {
+        sql << "create table soci_test(id integer primary key, val integer)";
+    }
+};
+
+TEST_CASE("SQLite row int64", "[sqlite][row][int64]") {
```

Comment:
Minor style nitpick:
```suggestion
TEST_CASE("SQLite row int64", "[sqlite][row][int64]")
{
```